### PR TITLE
Upgrade ChartJS version to 3.9.1

### PR DESCRIPTION
**Pull request by fjaguado**

### CODE REVIEW

File: projects/showcase/src/app/components/chart/showcase-chart.component.ts [hunk 1]
```diff
@@ -265,6 +242,7 @@ export class ShowcaseChartComponent {
 				fill = true;
 			}
 			this.dataLine.push(new ChartItem('Line ' + h, dataRnd, '', '', fill, true, false, 3));
+			console.log({rnd, dataRnd});
```
Comment (oGalceran1) [inline projects/showcase/src/app/components/chart/showcase-chart.component.ts:245]: I think that a console.log should not be present in a library

File: projects/systelab-charts/.eslintrc.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
       ],
       "parserOptions": {
         "project": [
-          "projects/systelab-charts/tsconfig.lib.json",
-          "projects/systelab-charts/tsconfig.spec.json"
+          "tsconfig.lib.json",
```
Comment (oGalceran1) [inline projects/systelab-charts/.eslintrc.json:13]: Is this correct or is it better to use ./tsconfig.lib.json ?

**Review by oGalceran1 — COMMENTED** [review]
Just a couple of comments.
Good work !!!

File: projects/systelab-charts/package.json [hunk 3]
```diff
@@ -13,14 +13,16 @@
...
   "peerDependencies": {
     "@angular/common": ">=15.0.0 <16.0.0",
     "@angular/core": ">=15.0.0 <16.0.0",
-    "tslib": "^2.1.0",
-    "chart.js": "^2.9.3",
-    "chartjs-plugin-annotation": "^0.5.7",
-    "chartjs-plugin-datalabels": "^0.7.0",
-    "date-fns": "^2.28.0",
-    "chartjs-adapter-date-fns": "^2.0.0"
+    "chart.js": "^3.9.1",
```
Comment (joaquimvila) [inline projects/systelab-charts/package.json:22]: Shouldn't we define a range of chart.js libraries to be provided? 
That would clarify which is the expectation of libs to be injected.

**Review by joaquimvila — APPROVED** [review]
Good job @fjaguado! Thanks for this version upgrade. Just a minor comment about how peerDependencies should be defined.

File: projects/systelab-charts/.eslintrc.json (hunk 2 above)
Comment (fjaguado) [reply projects/systelab-charts/.eslintrc.json:13]: I'll change to ./tsconfig.lib.json

**Review by oGalceran1 — APPROVED** [review]
OK

**Review by joaquimvila — APPROVED** [review]

**Pull request merged**